enrolments/admin.py

```diff
@@ -1,3 +1,15 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Enrolment
+
+
+class EnrolmentAdmin(admin.ModelAdmin):
+    # add class once model is created
+    list_display = ("id", "family")
```

Comment:
I'm imagining an admin would also like to see when the enrolment was created, when it was updated, and perhaps family surname if that's possible.

Reply:
good call! we should add `created_at` and `updated_at` for now just for testing
we don't have family surnames yet (incoming on student model), but that'll be useful for us to add